project/Dependencies.scala

```diff
@@ -1,5 +1,6 @@
 import sbt._
 
+//noinspection TypeAnnotation
```

Comment:
What does this do?

Reply:
It disables the 'Type annotation required' inspection for the class. I really don't think it's necessary on the Dependencies.